cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
-            image will be downsampled into a 250x300 image if a factor of 2
-            is entered.""" % globals())
+            "Image resampling factor", 2, minval=1, doc="""    
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method and
+spline parameters are not calculated automatically)*
+This setting controls how the image is resampled to make a smaller
+image. Resampling will speed up processing, but may degrade performance
+if the resampling factor is larger than the diameter of foreground
+objects. The image will be downsampled by the factor you enter. For
+instance, a 500x600 image will be downsampled into a 250x300 image if a
+factor of 2 is entered.
+""" % globals())
 
         self.spline_maximum_iterations = cps.Integer(
-                "Maximum number of iterations", 40, minval=1, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting determines the maximum number of iterations
-            of the algorithm to be performed. The algorithm will perform
-            fewer iterations if it converges.""" % globals())
+            "Maximum number of iterations", 40, minval=1, doc="""
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method and
+spline parameters are not calculated automatically)*
+This setting determines the maximum number of iterations of the
+algorithm to be performed. The algorithm will perform fewer iterations
+if it converges.
+""" % globals())
 
         self.spline_convergence = cps.Float(
-                "Residual value for convergence", value=.001, minval=.00001, maxval=.1, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting determines the convergence criterion.
-            The software sets the convergence criterion to the number entered
-            here times the signal intensity; the convergence you enter is the
-            fraction of the signal intensity that indicates convergence.
-            The algorithm derives a standard deviation of the background
-            pixels from the calculated background on each iteration. The
-            algorithm terminates when the difference between the standard
-            deviation for the current iteration and the previous iteration
-            is less than the convergence criterion.
-            <p>Enter a smaller number for the convergence to calculate a
-            more accurate background. Enter a larger number to calculate
-            the background using fewer iterations, but less accuracy.""" % globals())
+            "Residual value for convergence", value=.001, minval=.00001, maxval=.1, doc="""           
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.